hackathon_site/dashboard/frontend/src/components/dashboard/ItemTable/ItemTable.test.js

```diff
@@ -142,6 +150,21 @@ describe("Connected tables", () => {
         );
     });
 
+    it("Calls reportIncident when the 'Report broken/lost' button is clicked", () => {
+        const oneRow = [
+            { id: 1, url: "https://i.imgur.com/IO6e5a6.jpg", name: "Arduino", qty: 6 },
+        ];
+        const reportIncidentSpy = jest.fn();
+        const { getByText } = render(
+            withStore(
+                <CheckedOutTable items={oneRow} reportIncident={reportIncidentSpy} />
```

Comment:
We should also test that `push` was called. Two ways you could do that:
- Actually pass in a mock store to `withStore`, and check that the push action was dispatched.
- Use `UnconnectedCheckedOutTable` instead, and pass in a spy for `push` as well as `reportIncident` (and move it to the right `describe()` section of the test). Would recommend doing that, since there's no need for this test to be on the connected component.